src/status_im/contexts/chat/messenger/composer/mentions/style.cljs

```diff
@@ -2,29 +2,38 @@
   (:require
     [quo.foundations.colors :as colors]
     [react-native.platform :as platform]
-    [react-native.reanimated :as reanimated]
     [status-im.contexts.chat.messenger.composer.constants :as constants]))
 
-(defn shadow
-  [theme]
-  (if platform/ios?
-    {:shadow-radius  (colors/theme-colors 30 50 theme)
-     :shadow-opacity (colors/theme-colors 0.1 0.7 theme)
-     :shadow-color   colors/neutral-100
-     :shadow-offset  {:width 0 :height (colors/theme-colors 8 12 theme)}}
-    {:elevation 10}))
+(defn- add-shadow
+  [theme styles]
+  (cond-> styles
+    platform/ios? (assoc :shadow-radius (colors/theme-colors 30 50 theme)
+                         :shadow-opacity (colors/theme-colors 0.1 0.7 theme)
+                         :shadow-color colors/neutral-100
+                         :shadow-offset {:width 0 :height (colors/theme-colors 8 12 theme)})
+    :else (assoc :elevation 10)))
+
+(defn inner
+  [top theme]
+  (add-shadow theme
+              {:position         :absolute
+               :top              (- (+ 8 top))
+               :left             8
+               :right            8
+               :border-radius    16
+               :z-index          4
+               :max-height       constants/mentions-max-height
+               :background-color (colors/theme-colors colors/white colors/neutral-95 theme)}))
 
 (defn container
   [opacity top theme]
-  (reanimated/apply-animations-to-style
-   {:opacity opacity}
-   (merge
-    {:position         :absolute
-     :top              (- (+ 8 top))
-     :left             8
-     :right            8
-     :border-radius    16
-     :z-index          4
-     :max-height       constants/mentions-max-height
-     :background-color (colors/theme-colors colors/white colors/neutral-95 theme)}
-    (shadow theme))))
+  [{:opacity opacity}
+   (add-shadow theme
+               {:position         :absolute
+                :top              (- (+ 8 top))
+                :left             8
+                :right            8
+                :border-radius    16
+                :z-index          4
+                :max-height       constants/mentions-max-height
+                :background-color (colors/theme-colors colors/white colors/neutral-95 theme)})])
```

Comment:
I tried severalways to solve the rendering issue in the mentions, but nothing else worked. 

The problem is caused by our `apply-animations-to-style` wrapper, seems its usage delays the rendering of its stytles ~1 frame.

By avoiding its usage, we get the component state updated on time, here we are using the vector syntax introduced a while ago.

CC: @ilmotta 

By using the inline reanimated styles,